src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carDayRent = 40;
+
+  if (days >= 3 && days < 7) {
+    return (days * carDayRent) - 20;
+  } else if (days >= 7) {
+    return (days * carDayRent) - 50;
+  } else {
+    return days * carDayRent;
+  }
```

Comment:
last else block is unnecessary 
```suggestion
  }
  return days * carDayRent;
 
```